projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -146,7 +154,16 @@ export class ExplorerComponent {
     this.vizDashboard$ = this.explorerDashboardBuilder.visualizationDashboard$;
     this.initialState$ = activatedRoute.queryParamMap.pipe(
       take(1),
-      map(paramMap => this.mapToInitialState(paramMap))
+      map(paramMap => this.mapToInitialState(paramMap)),
+      tap(initialState => {
+        // Updating initial visualization and results expanded state
+        this.visualizationExpanded = initialState.visualizationExpanded ?? true;
+        this.resultsExpanded = initialState.resultsExpanded ?? true;
+        this.navigationService.addQueryParametersToUrl({
```

Comment:
I think there was a misunderstanding of the ask here - the expand/collapse shouldn't be in the URL, it should be persisted as a preference (see usages of preference service), similar to how table sorts and columns work.

Reply:
Can someone edit these preferences?

Reply:
Currently, they're only indirectly exposed. A user's actions update their preferences. Eventually, some may be put in front of a UI (not things like expand collapse but formats, colors etc.)

The simplest example of interaction with preferences is probably the expand/collapse behavior of the left nav in `hypertrace-ui/src/app/shared/navigation/navigation.component.ts`

Reply:
Done!!